City-of-Helsinki/helsinki-design-system
# feat: add technology tags to documentation

## Description

- Technology tags to all components in the documentation (on Components overview page)
- Add tags to the scaffolding script

# To think
- where Vanilla JS link (on Components overview page) should go?

## Related Issue

Closes [HDS-1366](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1366)

## Motivation and Context

It would be user friendly if there would be component tags (as Stable, Accessible, etc) that include the packages that it’s in.

## How Has This Been Tested?

- See it in the component overview page in the documentation.
- Try scaffolding script to chose the core tag (React tag is added automatically)

## Demos:

Links to demos are in the comments

## Screenshots (if appropriate):

## Add to changelog

- [x] Added needed line to changelog



[HDS-1366]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-1366?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
